src/inputHandler.ts

```diff
@@ -31,9 +31,13 @@ interface sqsRepoMessage {
   installation_id: number
 }
 
-export async function httpQueryRepoHandler (event: APIGatewayProxyEventV2): Promise<APIGatewayProxyResultV2> {
+export async function httpQueryRepoHandler(event: APIGatewayProxyEventV2): Promise<string | undefined> {
+  if (!event.pathParameters) {
+    throw 'Not sure what to do here.'
```

Comment:
this shouldn't be possible, lambda+api gateway routing shouldn't get it here without the path parameters, it'll also probably always be an array even if empty

Reply:
The types suggest that you can. If you say it shouldn't be, just use `event.pathParameters!.something`